security/acl.rst

```diff
@@ -21,7 +21,7 @@ the ACL system comes in.
     class will handle the logic behind the scenes, instead of the ACL system.
 
 Imagine you are designing a blog system where your users can comment on your
-posts. Now, you want a user to be able to edit their own comments, but not those
+posts. Now, you want an user to be able to edit their own comments, but not those
```

Comment:
Choosing between "a" and "an" does not depend on the first character of the following word to be a vocal, but depends on how it is pronounced. Thus, using "a" here is correct.